onnxruntime/core/framework/tensorprotoutils.h

```diff
@@ -76,6 +76,13 @@ ONNXTensorElementDataType GetTensorElementType(const ONNX_NAMESPACE::TensorProto
 template <size_t alignment>
 common::Status GetSizeInBytesFromTensorProto(const ONNX_NAMESPACE::TensorProto& tensor_proto, size_t* out);
 
+/**
+Special marker used to indicate an existing memory buffer contains the TensorProto external data.
+If the 'location' field of the external data info is set to this marker, the 'offset' field should contain the
+address of the memory containing the data.
+*/
+constexpr const ORTCHAR_T* kMemoryAddressTag = ORT_TSTR("__ORT_MEM_ADDR__");
```

Comment:
>ORTCHAR_T [](http://example.com/codeflow?start=16&length=9)

Any reason this should be UTF-16? I would think we always store it in UTF-8.

Reply:
We read the location field from the external data using [ToWideString](https://github.com/microsoft/onnxruntime/blob/37995a7245895c1116e67a90d72c4c568dac8282/onnxruntime/core/framework/tensor_external_data_info.cc#L26) and the comparison is with that value.

Reply:
is it possible to pick a value that's not a valid file path?

Reply:
definitely